src/constants/usdPriceToken.ts

```diff
@@ -0,0 +1,9 @@
+import { Token } from 'types';
+
+// by convention, USD prices are informed with 18 decimals
+export const USD_PRICE_TOKEN: Token = {
+  decimals: 18,
+  address: '',
+  asset: '',
+  symbol: '',
+};
```

Comment:
This feels risky: someone in the future could use it as a reference for a "USD" token.
I understand this is to be able to use `convertPriceMantissaToDollars` in order to convert USD prices expressed with 18 decimals. I'd recommend updating `convertPriceMantissaToDollars` instead, so that instead of taking a `token` field it takes a `decimals` field. This function doesn't need to receive the full token definition since we only use the decimals for the calculation.